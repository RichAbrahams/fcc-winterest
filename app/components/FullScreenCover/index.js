/**
*
* FullScreenCover
*
*/

import React from 'react';
import { Wrapper } from './styles';


function FullScreenCover(props) {
  return (
    <Wrapper>
      {props.children}
    </Wrapper>
  );
}

FullScreenCover.propTypes = {
  children: React.PropTypes.object,
};

export default FullScreenCover;
