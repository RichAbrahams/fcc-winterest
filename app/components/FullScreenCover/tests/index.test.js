import React from 'react';
import renderer from 'react-test-renderer';
import FullScreenCover from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FullScreenCover>test</FullScreenCover>).toJSON();
  expect(tree).toMatchSnapshot();
});
