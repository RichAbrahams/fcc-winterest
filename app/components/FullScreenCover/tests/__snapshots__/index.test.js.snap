exports[`test renders correctly 1`] = `
<div
  className="ctDVNg">
  test
</div>
`;
